data/drak/indigenous.txt

```diff
@@ -125,7 +125,7 @@ ship "Void Sprite" "Void Sprite (Infant)"
 	thumbnail "thumbnail/void sprite infant"
 	"never disabled"
 	attributes
-		category "Light Warship"
+		category "Unclassified Minor"
```

Comment:
This isn't a carried unit, so should probably be `unclassified` rather than `unclassified minor`.

Reply:
It's an infant, which is to say minor, void sprite. Even if there's currently no void sprites that carry it, it still felt appropriate in terms of lore to name it as such.

Reply:
Categories are meant to be mechanical terms of a ship's purpose - i.e., utility, heavy warship, etc., - they aren't necessarily a reference to a ship's lore, but to its functionality. So in this case, for mechanical consistency, if we are to have a `minor` tag for `unclassified`, it makes sense that it should be tied to a difference in function, not lore, which would be the ability to be carried in this case.

Reply:
If `unclassified minor` is meant to be specifically relative to a ships' (or species') size or growth compared to others, it would make more sense that there is an overall notable amount of lifeforms like this, though I don't think there are enough to warrant that distinction, and ultimately, the `minor` is necessary because the carriable Jje need to be a separate category anyways.

Otherwise, it makes sense that any non "-ship" would just be `Unclassified`. It seems odd to have multiple types of "Unclassification" for something that is a result of ambiguity - if we saw to different aliens in real life, say, one large. "old" and large in numbers, and one small and "young", I'd imagine we just call them "aliens", not "alien minor" and "alien"; the `minor` designation is more a result of a necessary functionality.